assets/styles.css

```diff
@@ -0,0 +1,200 @@
+* {
+  box-sizing: border-box;
+  font-family: "Josefin Sans", sans-serif;
+  margin: 0;
+  padding: 0;
+}
+
+nav li,
+a {
+  font-family: "Josefin Sans", sans-serif;
+  font-weight: 500;
+  font-size: x-large;
+  color: #d9d9d9;
+  text-decoration: none;
+}
+
+header {
+  display: flex;
+  justify-content: flex-end;
+  align-items: center;
+  padding: 5px 10%;
+  background-color: #2e2e2e;
+  border-radius: 0.5rem;
+}
+
+.nameLogo {
+  margin-right: auto;
+  /* cursor: pointer; */
+}
+
+.nav-container li {
+  display: inline-block;
+  padding: 0px 20px;
+}
+
+.nav-container li a {
+  transition: all 0.3s ease 0s;
+}
+
+.nav-container li a:hover {
+  color: #2d7eb4;
+}
+
+@media screen and (max-width: 576px) {
+  .nav-container {
+    flex-direction: column;
+    align-items: center;
+  }
+
+  .nav-item {
+    padding: 10px;
+  }
+}
+
+.banner-area {
+  background-image: url("../images/long-banner-blackgold.jpg");
+  background-size: cover;
+  background-position: center center;
+  top: 100px;
+  height: 250px;
+  width: 100%;
+  margin-top: 3px;
+  position: inherit;
+  border-radius: 0.5rem;
+}
+
+.portfolio-portrait {
+  top: 10px;
+  height: 30%;
+  width: 30%;
+  padding-right: 2%;
+  position: inherit;
+}
+
+.about-me {
+  display: flex;
+  flex-direction: row;
+  margin-top: 2%;
+  font-family: "Josefin Sans", sans-serif;
+  font-size: large;
+  color: #353535;
+  text-decoration: none;
+  padding-left: 1%;
+}
+
+.about-me h2 {
+  font-size: xx-large;
+  margin-bottom: 30px;
+  margin-top: 10px;
+}
+
+/* Variable colors */
+:root {
+  --dark: #353535;
+  --light: #d9d9d9;
+  --purple: #2d7eb4;
+}
+
+/* card wrapper */
+.wrapper {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+}
+
+/* CSS for cards styling below */
+body {
+  margin: 2rem;
+  color: var(--dark);
+}
+
+.card {
+  overflow: hidden;
+  background-color: #f5f5f5;
+  box-shadow: 0 10px 10px rgb(90, 90, 90);
+  border: 3px solid var(--dark);
+  border-radius: 0.5rem;
+  position: relative;
+  width: 350px;
+  margin: 1rem;
+  transition: all 0.3s ease-in-out;
+  cursor: pointer;
+}
+
+.card:hover {
+  transform-style: (1.05);
+  transform: scale(1.1);
+  box-shadow: 0px 2px 40px var(--purple);
+}
+
+/* larger top project card */
```

Comment:
I would like to see more comments in this file to explain what section of the HTML code we are styling. You don't need to comment each selector block, but once you've grouped the ones together that match up with a certain HTMLsection, you can add a group comment